internal/proxy/proxy.go

```diff
@@ -83,9 +84,13 @@ type InstanceConnConfig struct {
 	// Port is the port on which to bind a listener for the instance.
 	Port int
 	// UnixSocket is the directory where a Unix socket will be created,
-	// connected to the Cloud SQL instance. If set, takes precedence over Addr
-	// and Port.
+	// connected to the Cloud SQL instance. The full path to the socket will be
+	// UnixSocket + "/" + Name. If set, takes precedence over Addr and Port.
```

Comment:
On windows, the forward slash isn't in the name. Let's either remove the sentence or update it to be accurate, e.g., The socket will be created in the UnixSocket directory with Name. When connecting to Postgres, Name will be a directory, etc etc.